src/Microsoft.Azure.WebJobs.Extensions.Dapr/Utils/EnvironmentExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace Microsoft.Azure.WebJobs.Extensions.Dapr.Utils
+{
+    using System;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Extensions.Logging;
+
+    internal static class EnvironmentExtensions
```

Comment:
nit: this class sounds like an extension method class https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/extension-methods, maybe we can call it `EnvironmentUtils`.